### DISCUSSION TITLE
Portrait mode on a droid blue background

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#19] [General]
@brianzelip 

When I toggle to portrait mode on my mobile on the homepage, the swinging bridge picture goes away and I get this blue backdrop 
![Screenshot_20240312-163018.png](https://github.com/Relay-Maryland/relaymaryland.com/assets/11720131/7afd9826-a81d-42f4-a816-40ecb2b0e62d)

Just reporting and curious!

**Comment by brianzelip** [comment]
Yeah that’s expected. Could be any color, the image is so hard to see on smaller mobile that I felt it better to hide it since you couldn’t really tell what it was, and it was really busy in that section.
